docs/modules/concepts/pages/protocols/smtp.adoc

```diff
@@ -71,6 +71,67 @@ SMTP relates to:
 
 == Try It!
 
-Add a section to the demo, and connect manuall via an SMTP session
+You can connect manually to the James SMTP server using telnet.
```

Comment:
I don't really like that section: one should not run non-starttls servers and if you configure yours to use starttls you won't be able to test with telnet.
There's a solution with openssl to handle that situation but it makes things more complex than it's worth IMO.
What about providing a test tool instead?

Reply:
Note that this is *only* for the demo server, which should never be used in production.

Reply:
I could not find why it's demo server only from the structure or the text in this file.

Reply:
It is implied in the "Requirements" list: "Run the long demo (without stopping)". (That item still needs a link, of course.)

Can you suggest something else I could do to make it more obvious?

Reply:
Ok, I see now, I would say:

```suggestion
You can try speaking SMTP with your demo server by connecting to it using telnet. It would not work on a real world server as it would requires encrypted traffic. See "using openssl to talk to your server"
```

And I would write that section about using openssl (:

I can help if needed for that section, don't hesitate to ask